doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
 
 ## Table of Contents
 * [1. Overview](#1.0)
-* [2. Scope](#2.0)
-* [3. Principles](#3.0)
-* [4. Relevant Technologies](#4.0)
-  * [4.1 IO Virtualisation](#4.1)
-  * [4.2 SmartNICs](#4.2)
-  * [4.3 Acceleration Cards](#4.3)
-  * [4.4 FPGA](#4.4)
-  * [4.5 GPUs/NPUs](#4.5)
-  * [4.6 EPA/NFD](#4.6)
-* [5. CNTT Policies for Managing Non-Conforming Technologies ](#5.0) 
-  * [5.1 Feature Availability](#5.1)
-  * [5.2 Hardware-Dependent Coding](#5.2)
-  * [5.3 Current CNTT Policies](#5.3)
-
+    * [1.1 Glossary](#1.1)
+    * [1.2 Problem Statement](#1.2)
+    * [1.3 Project Goals and Purpose](#1.3)
+    * [1.4 Common NFVI Benefits](#1.4)
+* [2. Principles](#3.0)
+* [3. Scope](#3.0)
+  * [3.1 Specification Types](#3.1)
+  * [3.2 Bogo-Meter](#3.2)
+* [4. Approach](#4.0)
+* [5. Use Cases](#5.0)
+* [6. Roadmap and Releases](#6.0)
+* [7. CNTT Technical Policies and Transition Plan](#7.0)
+* [8. Relevant Technologies](#8.0)
 
 ## Available Specifications
 * [Reference Model](../ref_model)
 * [Reference Architecture](../ref_arch)
 * [Reference Implementation](../ref_impl)
 * [Reference Conformance](../ref_cert)
 
-## CNTT Releases
-* [CNTT Releases](./release_notes)
-* [CNTT Roadmap](./roadmap.md#2)
-
 <a name="1.0"></a>
-## 1. Overview
+# 1. Overview
 
 Initially organized early in 2019, the Common Network Function Virtualization Infrastructure Telecom Taskforce (CNTT) was created in response to rapid changes in how networking applications are being designed, built and managed, plus a growing recognition of a perceived functional gap between the previous standard infrastructure models and the architectures needed to support Network Function Virtualization (NFV) applications.  Organizationally the Common Telco Network Function Virtualization Infrastructure (NFVI)  project, jointly hosted by GSMA and the Linux Foundation, operates as an open committee responsible for creating and documenting an industry aligned Common NFVI Framework.  The CNTT was created with the intent that it would create the NFVI framework, and eventually morph into an on-going project under the auspices of the GSMA and the Linux Foundation umbrellas.  The final on-going operational form of the Taskforce will be determined as the project evolves. 
```

Comment:
Given the discussion in #1168 can I suggest something along the lines of the below to be added to this section.

> As part of the Baldy release, work to ensure the Reference Model is agnostic across VM-based and container-based workloads and infrastructure took place, with references to "NFVI" changing to "Cloud Infrastructure" to ensure the Reference Model is not too VM-based in it's approach and terminology used.

Reply:
added one line to RM intro to incorporate this in here: https://github.com/cntt-n/CNTT/pull/1267/commits/1d599bfd1f56b482913d3f6b4c84328331931433